plugins/discourse-calendar/spec/services/discourse_post_event/chat_channel_sync_spec.rb

```diff
@@ -27,4 +27,11 @@
 
     expect(event.chat_channel_id).to be_nil
   end
+
+  it "defaults event name to post title" do
+    post = Fabricate(:post, user: Fabricate(:admin))
+    event = Fabricate(:event, chat_enabled: true, post: post)
```

Comment:
you could reuse the top-level fab I think

```suggestion
    event = Fabricate(:event, chat_enabled: true, post: admin_post)
```